frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/gitrepositoryselection/GitRepositoryComboBox.java

```diff
@@ -56,8 +56,6 @@ public MutableCollectionComboBoxModel<GitRepository> getModel() {
 
   @UIEffect
   private void updateRepositories() {
-    // A bit of a shortcut: we're accessing filesystem even though we are on the UI thread here;
```

Comment:
These comments are still valid; it's just that now all such cases are "managed centrally" via ArchUnit